static/styles/tooltips.css

```diff
@@ -4,6 +4,7 @@
      * for tooltips here.
      */
     font-family: "Source Sans 3", sans-serif !important;
+    font-feature-settings: "ss01" on;
```

Comment:
Can you extend the commit message explaining why we want this? Maybe also worth adding a comment explaining what it does, since this isn't a CSS feature I'm familiar with off the top of my head, and that's probably going to be true for other readers.

Reply:
@terpimost could you please elaborate on this one, cause I don't quite understand the details of it either.